website/docusaurus.config.ts

```diff
@@ -27,6 +36,10 @@ const config: Config = {
     locales: ['en'],
   },
 
+  customFields: {
+    storybookBaseUrl: storybookBaseUrl,
```

Comment:
me being annoying and nitty
```suggestion
    storybookBaseUrl,
```